tests/wpunit/Migrations/FeePlanMigrationTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace Migrations;
+
+use Codeception\Util\Debug;
+use WPBDP__Fee_Plan;
+
+class FeePlanMigrationTest extends \Codeception\Test\Unit {
+
+	/**
+	 * @var \WpunitTester
+	 */
+	protected $tester;
+
+	protected function _before() {
+		$this->save_fee();
+	}
+
+	protected function _after() {
+		global $wpdb;
+
+		// Remove any existing payment in database
+		$wpdb->query( "UPDATE {$wpdb->prefix}wpbdp_plans SET `enabled` = 1 WHERE `amount` > 0;" );
+	}
+
+	public function testMigration() {
+		global $wpdb;
+		$this->tester->wantToTest( 'Migration' );
+
+		require_once( WPBDP_PATH . 'includes/admin/upgrades/migrations/migration-18_5.php' );
+		
+		$all         = absint( $wpdb->get_var( "SELECT COUNT(*) FROM {$wpdb->prefix}wpbdp_plans" ) );
+		$disabled    = absint( $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM {$wpdb->prefix}wpbdp_plans WHERE enabled = %d", 0 ) ) );
+		$enabled     = $all - $disabled;
+
+		Debug::debug( 'Total disabled ' . $disabled );
+		Debug::debug( 'Total enabled ' . $enabled );
+
+		$migrator    = new WPBDP__Migrations__18_5();
+		$migrator->migrate();
+		$after_disabled    = absint( $wpdb->get_var( $wpdb->prepare( "SELECT COUNT(*) FROM {$wpdb->prefix}wpbdp_plans WHERE enabled = %d", 0 ) ) );
+		$after_enabled     = $all - $after_disabled;
```

Comment:
I don't see any checks to compare actual and expected results here.

Reply:
The logic on this is still being worked on and the test case on this is skipped based on other issues

Reply:
Hey @stephywells I have been doing some research on the automated code tests. Generally the error I am facing is this https://githubmemory.com/repo/lucatume/wp-browser/issues/497 . From the CodeCeption Lucatime dev, the resolution is https://wpbrowser.wptestkit.dev/faq#can-i-run-all-my-tests-with-one-command . I have modified the codeception script to run each wpunit file separately for the tests. Running in parallel causes the database connection to be lost as it creates and drops WordPress data on each  test